cpp/proto/substrait/extension_rels.proto

```diff
@@ -69,3 +69,4 @@ message SegmentedAggregateRel {
   // A list of one or more aggregate expressions along with an optional filter.
   repeated substrait.AggregateRel.Measure measures = 3;
 }
+
```

Comment:
Nit: revert this change? (Since we don't touch this file at all probably cleaner if we don't leave a new line change)